components/Dropdown/Dropdown.less

```diff
@@ -36,6 +37,7 @@
     }
   }
   .dropdown-hidden {
+    opacity: 0;
     display: none;
   }
```

Comment:
Changing the `display` property will interfere with `transition`-based animations. Because removing the `"dropdown-hidden"` class will cause your dropdown menu to change its `display` value from `none` to `flex`, this will cancel out the fade-in animation that you intended to set up.

A possible fix would be to use `visibility: hidden;` here instead.